lib/streamlit/media_file_manager.py

```diff
@@ -58,8 +63,19 @@ def _calculate_file_id(data, mimetype):
     return filehash.hexdigest()
 
 
+def _get_extension_for_mimetype(mimetype: str) -> str:
+    try:
+        return PREFERRED_MIMETYPE_EXTENSION_MAP[mimetype]
+    except KeyError:
+        extension = _mimetypes.guess_extension(mimetype)
+        if extension is None:
+            return ""
+
```

Comment:
This is not a big deal, but I would write this a little differently to avoid exception handling.

```python
if mimetype in PREFERRED_MIME_TYPE_EXTENSION_MAP:
    return PREFERRED_MIMETYPE_EXTENSION_MAP[mimetype]

extension = _mimetypes.guess_extension(mimetype)
if extension is None:
        return ""

return extension
```